src/MoBi.Core/Serialization/ORM/Mappers/ProjectMetaDataToProjectMapper.cs

```diff
@@ -106,7 +106,7 @@ private void addBuildingBlocksToModule(List<IBuildingBlock> moduleBuildingBlocks
 
       private void addSimulationSettingsBuildingBlocks(List<SimulationSettings> simulationSettingsBlocks)
       {
-         if (simulationSettingsBlocks.Count == 1)
+         if (simulationSettingsBlocks.Any() && _project.SimulationSettings == null)
```

Comment:
We had this as a TODO but I think this is as good a solution as any, so maybe we can just remove the TODO comment from below (all the commented code below) and we can call this TODO -> DONE

Or if someone has a better plan for this conversion. This does not change the simulation settings, only sets the default for new simulations created in V12 or later.

Reply:
no better plan :)